sample-input/point-source/point-source.py

```diff
@@ -0,0 +1,145 @@
+
+
+###############################################################################
+#                          Main Simulation Parameters
+###############################################################################
+
+options = Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+log.set_log_level('NORMAL')
+
+
+###############################################################################
+#                            Creating Materials
+###############################################################################
+
+log.py_printf('NORMAL', 'Importing materials data from HDF5...')
+
+materials = materialize.materialize('../c5g7-materials.h5')
+
+
+###############################################################################
+#                            Creating Surfaces
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating surfaces...')
+
+left = XPlane(x=-20.0, name='left')
+right = XPlane(x=20.0, name='right')
+top = YPlane(y=20.0, name='top')
+bottom = YPlane(y=-20.0, name='bottom')
+boundaries = [left, right, top, bottom]
+
+for boundary in boundaries: boundary.setBoundaryType(VACUUM)
+
+
+###############################################################################
+#                             Creating Cells
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating cells...')
+
+water_cell = CellBasic(name='water')
+water_cell.setMaterial(materials['Water'])
+
+source_cell = CellBasic(name='source')
+source_cell.setMaterial(materials['Water'])
+
+root_cell = CellFill(name='root cell')
+root_cell.addSurface(halfspace=+1, surface=left)
+root_cell.addSurface(halfspace=-1, surface=right)
+root_cell.addSurface(halfspace=+1, surface=bottom)
+root_cell.addSurface(halfspace=-1, surface=top)
+
+
+###############################################################################
+#                            Creating Universes
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating universes...')
+
+water_univ = Universe(name='water')
+source_univ = Universe(name='source')
+root_universe = Universe(name='root universe')
+
+water_univ.addCell(water_cell)
+source_univ.addCell(source_cell)
+root_universe.addCell(root_cell)
+
+
+###############################################################################
+#                            Creating Lattices
+###############################################################################
+
+num_x = 200
+num_y = 200
+width_x = (root_universe.getMaxX() - root_universe.getMinX()) / num_y
+width_y = (root_universe.getMaxY() - root_universe.getMinY()) / num_x
+universes = [[water_univ]*num_x for _ in range(num_y)]
+universes[15][15] = source_univ
```

Comment:
Change the placement of ``source_univ``? Do you mean something like:

```python
universes[num_x/2][num_x/2] = source_univ
```

I can see how this would make the input file work seamlessly with more/less mesh. My main intention was to offset the fixed source so that it wouldn't be centered in the lattice. So if the above is what you were recommending, I can come up with a way to combine both.